packages/atomic/src/components/search/result-template-components/atomic-quickview-modal/atomic-quickview-modal.tsx

```diff
@@ -118,36 +133,25 @@ export class AtomicQuickviewModal implements InitializableComponent {
   }
 
   private renderFooter() {
-    const quickviewsInfoFromResultList =
-      this.bindings.store.get('resultList')?.quickviews;
-    const currentQuickviewPosition = this.bindings.store.get(
-      'currentQuickviewPosition'
-    );
-
-    const first =
-      (quickviewsInfoFromResultList?.position[currentQuickviewPosition] || 0) +
-      1;
-    const total = quickviewsInfoFromResultList?.total;
-
     return (
       <div slot="footer" class="flex items-center gap-2">
         <Button
           class="p-2"
           style="square-neutral"
-          onClick={() => this.bindings.store.previousQuickview()}
-          text="Prev"
+          onClick={() => this.previousQuickview?.emit()}
+          text={this.bindings.i18n.t('previous')}
```

Comment:
I'm not sure if I already mentioned it, but when we do accessibility for this, we should probably make the `previous`/`next` labels more descriptive for these buttons, like "Previous quickview"/"Next quickview". 

Alternatively, [WCAG technique H71](https://www.w3.org/TR/WCAG20-TECHS/H71.html) specifies that we can use a fieldset/legend to group related form controls (our utility component could take care of removing unnecessary style), but I'm not sure if previous/next buttons would be considered form controls.